server/app-engine/static/graas.js

```diff
@@ -90,6 +87,19 @@ function getDayOfWeek() {
     } else return ((new Date()).getDay() + 6) % 7;
 }
 
+// returns date as an 8-character string (ie 20220317 for 3/17/22)
+function getDate() {
+    if (testDate) {
+        return testDate;
+    } else {
+        var today = new Date();
+        var year = today.getFullYear();
+        var month = today.getMonth() + 1;
+        var date = today.getDate();
+        return (year * 10000) + (month * 100) + date;
+    }
+}
+
```

Comment:
gtfs-rt-util.js has some time and date utility functions. Consider adding this there.

Reply:
updated - mind taking one more quick look? Tested and is working fine